Dockerfile

```diff
@@ -29,4 +29,4 @@ RUN compass compile
 
 # Expose port 80 and run the application.
 EXPOSE 80
-CMD ["/home/bottle/wireless-debugging-master/server/widb_server.py"]
+CMD ["/home/wirelessdebug/server/widb_server.py"]
```

Comment:
No, this has to be absolute. CMD is executed after the setup process and WORKDIR does not apply. (It basically is run on a different VM. Actually, it's basically black magic.) I'm going to change it to ENTRYPOINT though since CMD seems to be more for passing parameters to the program.